pandas/plotting/_core.py

```diff
@@ -507,86 +507,76 @@ class PlotAccessor(PandasObject):
     Parameters
     ----------
     data : Series or DataFrame
-        The object for which the method is called
+        The object for which the method is called.
     x : label or position, default None
         Only used if data is a DataFrame.
     y : label, position or list of label, positions, default None
         Allows plotting of one column versus another. Only used if data is a
         DataFrame.
-    kind : str
-        The kind of plot to produce:
-
-        - 'line' : line plot (default)
-        - 'bar' : vertical bar plot
-        - 'barh' : horizontal bar plot
-        - 'hist' : histogram
-        - 'box' : boxplot
-        - 'kde' : Kernel Density Estimation plot
-        - 'density' : same as 'kde'
-        - 'area' : area plot
-        - 'pie' : pie plot
-        - 'scatter' : scatter plot
-        - 'hexbin' : hexbin plot
+    kind : {'line', 'bar', 'barh', 'hist', 'box', 'kde', 'density', 'area', \
```

Comment:
Do you mind reverting this one? We'll change the guidelines, I think the previous is more clear.

Reply:
yep, i think this is the same scenario as #28739 , regarding long lists of values